docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -2523,4 +2523,134 @@ Enabled JSON-RPC APIs.
+'{"jsonrpc":"2.0","method":"eea_sendRawTransaction","params": [{see
+above}], "id":1}' <JSON-RPC-http-endpoint:port> ```
+  
+   ```bash tab="wscat WS request"
+   {"jsonrpc":"2.0","method":"eea_sendRawTransaction","params": [{see above}], "id":1}
+   ```
+  
+   ```json tab="JSON result"
+   {
+     "id":1,
+     "jsonrpc": "2.0",
+     "result": "0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331"
+   }
+   ```
+
+### eea_getTransactionReceipt
+
+Returns the contract address, after the transaction was mined. Receipts for pending transactions
+are not available.
+
+**Parameters**
+
+`DATA` - 32-byte hash of a transaction.
+
+**Returns**
+
+`Object` - [Transaction receipt object](JSON-RPC-API-Objects.md#transaction-receipt-object), or `null` when no receipt is found.
+
+!!! example ```bash tab="curl HTTP request" $ curl -X POST --dataFLRY3
+'{"jsonrpc":"2.0","method":"eea_getTransactionReceipt","params":["0x504ce587a65bdbdb6414a0c6c16d86a04dd79bfcc4f2950eec9634b30ce5370f"],"id":53}'
+<JSON-RPC-http-endpoint:port> ```
+  
+   ```bash tab="wscat WS request"
+   {"jsonrpc":"2.0","method":"eea_getTransactionReceipt","params":["0x504ce587a65bdbdb6414a0c6c16d86a04dd79bfcc4f2950eec9634b30ce5370f"],"id":53}
+   ```
+  
+   ```json tab="JSON result"
+   {
+       "jsonrpc": "2.0",
+       "id": 1,
+       "result": {
+           "blockHash": "0xe7212a92cfb9b06addc80dec2a0dfae9ea94fd344efeb157c41e12994fcad60a",
+           "blockNumber": "0x50",
+           "contractAddress": null,
+           "cumulativeGasUsed": "0x5208",
+           "from": "0x627306090abab3a6e1400e9345bc60c78a8bef57",
+           "gasUsed": "0x5208",
+           "logs": [],
+           "logsBloom": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+           "status": "0x1",
+           "to": "0xf17f52151ebef6c7334fad080c5704d77216b732",
+           "transactionHash": "0xc00e97af59c6f88de163306935f7682af1a34c67245e414537d02e422815efc3",
+           "transactionIndex": "0x0"
+       }
+   }
+   ```
+
+
+### eea_clientCapabilities
+
+
+A call to `eea_clientCapabilities` provides more information about the
+capabilities supported by the client. This call returns the private
+transaction restriction levels and the kinds of consensus mechanisms
+supported.
+
+
+**Parameters**
+
+None.
+
+
+**Returns**
+
+This call returns client capability information fields in the format of [JSON] name values pairs:
+
+consensus : ["PoW", "IBFT" , "Raft"]
+restriction: ["restricted", "unrestricted"]
```

Comment:
Above it says `unrestricted` private transactions are not supported.  If that's correct, this methods would never return unrestricted would it?